### PR TITLE
Swift APIView: update MacOS deployment target

### PR DESCRIPTION
**Pull request by tjprescott**
Specifies minimum MacOS level 10.14 for supporting XCode 10.2.

### CODE REVIEW

File: src/swift/SwiftAPIView.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -421,6 +421,7 @@
 				DEVELOPMENT_TEAM = ZQB5E94758;
 				ENABLE_HARDENED_RUNTIME = YES;
 				INFOPLIST_FILE = "$(SRCROOT)/Info.plist";
+				MACOSX_DEPLOYMENT_TARGET = 10.15;
```
Comment (rasinive) [inline src/swift/SwiftAPIView.xcodeproj/project.pbxproj:424]: Note the value is set on project level, not target level.

File: src/swift/SwiftAPIView.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -435,6 +436,7 @@
...
```
Comment (rasinive) [inline src/swift/SwiftAPIView.xcodeproj/project.pbxproj:439]: Note the value is set on project level, not target level.

Comment (tjprescott) [reply src/swift/SwiftAPIView.xcodeproj/project.pbxproj]: Thanks @rasinive I updated them in both places to be consistent.

File: src/swift/SwiftAPIView.xcodeproj/project.pbxproj (hunk 1 above)
Comment (tjprescott) [reply src/swift/SwiftAPIView.xcodeproj/project.pbxproj]: Fixed.

**Review by rasinive — APPROVED** [review]

**Pull request merged**